### ISSUE TITLE
while opening the vega, it shows java was started but returned exit code =13

### ISSUE THREAD

**Issue by Saravana30** [subgraph/Vega#183]
![Vega_issue](https://user-images.githubusercontent.com/60920904/74227879-5fff2580-4ce5-11ea-9555-99dc518045df.PNG)

**Comment by Saravana30** [comment]
I have downloaded the 32bit, installed it. using windows 10

Kindly let me know how to solve this issue.

**Comment by anneborcherding** [comment]
Did you have a look into #165?

**Comment by Saravana30** [comment]
Yes.. But i'm using Vega 64 bit on JVM 64 bit. still facing same issue.

In additional, while opening the **I/O Exception trying to lock Vega instance: C:\Program Files(x86)\Vega\workspace\.metadata\.lock(Access is denied)**

**Comment by anneborcherding** [comment]
which java version are you using?

**Comment by Saravana30** [comment]
java version "1.8.0_241"
Java(TM) SE Runtime Environment (build 1.8.0_241-b07)
Java HotSpot(TM) 64-Bit Server VM (build 25.241-b07, mixed mode)

**Comment by Kino1994** [comment]
Run the java command shown in the photo with -verbose for get more information. Remember that paths with spaces must be quoted.

**Comment by Saravana30** [comment]
Can yo confirm below java command
java -verbose:

**Comment by Kino1994** [comment]
Hi @Saravana30

First of all:

You should have this configuration:

Vega Installation 64 bits
Java Installation 64 bits

Run in CMD as **ADMINISTRATOR** (if not will fails due to lack of permissions grant) this command (the command of your photo...). Remember that paths must be quoted. Check also the paths of your files in the command and if those files exists...

**"C:\Program Files (x86)\Common Files\Oracle\Java\javapath\javaw.exe"** -Xms256m -Xmx1024m -XX:PermSize=128m -XX:MaxPermSize=256m -jar "C:\Program Files\Vega\plugins\org.eclipse.eguinox.launcher_1.3.0.v20120522-1813.jar" -os win32 -ws win32 -arch x86 -showsplash -launcher "C:\Program Files\Vega\Vega.exe" -name Vega --launcher.library "C:\Program Files\Vega\plugins\org.eclipse.eguinox.launcher.win32.win32.x86_1.1.20 0.v20120522-1813\eclipse_1503.d11" -startup "C:\Program Files Wega\plugins \org.eclipse.eguinox.launcher_1.3.0.v20120522-1813.jar"  --launcher.overrideVmargs -exitdata 23e0_184 -vm "C:\Program Files (x86)\Common Files\Oracle\Java\javapath\javaw.exe" -vmargs -Xms256m -Xmx1024m -XX:PermSize=128m -XX:MaxPermSize=256m -jar "C:\Program Files\Vega\plugins\org.eclipse.eguinox.launcher_1.3.0.v20120522-1813.jar"

If fails change the argument -arch x86 to **-arch x86_64**. In my case it fails with -arch x86 because i have installed Vega 64 bits because I have a PC with 64 bits architecture. You say on 11 Feb that you installed the 32 bits version of Vega. If you changed to x86_64 and it works maybe is because you need to have a 64 bit Vega installation. In summary if you have a 64 bits PC you need a Vega Installation of 64 bits and also a JDK installation of 64 bits... If not, if you have a PC of 32 bits you need to have a installattion of 32 bits of Vega and JDK of 32 bits. Remember to refresh JAVA_HOME and PATH environmet variables. That could be a problem if you have multiple JDK installed.

In this case the thing that is happenning is that the JDK or Vega is passing the wrong architecture to the command:

https://stackoverflow.com/questions/21345903/java-was-started-but-returned-exit-code-13

Yo con see logs on your User eclipse path:

C:\Users\\**YourUser**\\.eclipse\com.subgraph.vega.application.product_1.0.0.201410142137_839234284\configuration

Keep in mind that my command is different from yours because I have JDK installed in other path...

![image](https://user-images.githubusercontent.com/49944060/78740107-2c8fee80-7956-11ea-8ff2-1e6585265d65.png)

You can also replace javaw.exe path **"C:\Program Files (x86)\Common Files\Oracle\Java\javapath\javaw.exe"** for **java -verbose** to run the command getting log information in CMD of how files are loaded and see also if you get an error

java -verbose -Xms256m -Xmx1024m .............

![image](https://user-images.githubusercontent.com/49944060/78737749-93aaa480-7950-11ea-8ecd-ccb1da0a6aa3.png)

I hope it helps you.
